core/src/operations/mod.rs

```diff
@@ -32,6 +32,10 @@ pub enum Operation {
     /// SYSCALL. Thus, this operation can be executed only inside a SYSCALL code block.
     Caller,
 
+    /// Pushes the current value of the current clock cycle onto the stack. This operation can be used
+    ///  to measure the number of cycles it has taken to execute the program up to the current instruction.
```

Comment:
nit: I think there is an extra "current" in the first sentence. Could probably be just "Pushes the current value of the clock cycle onto the stack."

Also, there is an extra space in the beginning of the second line.